### PR TITLE
Add subset lioness 

### PR DESCRIPTION
**Pull request by violafanfani**
- Fixed lioness start and end behavior (background stays the same)
- Added case where only a subset of samples is run ('1,4,5')
- Added a couple of tests to check this behavior
- Updated command line to follow behavior

### CODE REVIEW

**Comment by marouenbg** [comment]
@talkhanz  Now you can use subset_numbers in Lioness to give any sample id as input.

File: tests/test_lioness.py [hunk 1]
```diff
@@ -65,13 +67,32 @@ def test_lioness():
...
+    lioness_obj = Lioness(panda_obj,save_dir = "lioness_output", start=1, end = 2, save_single=True)
+    lioness_obj.save_lioness_results(output_results_npy_2)
+
+    # First, check that all save results are the same
+    res_npy2 = np.load(output_results_npy_2)
+    np.allclose(res_npy[:,:2], res_npy2)
+
+
+    # Check the same results for running only on sample 0 and 2
+    lioness_obj = Lioness(panda_obj,save_dir = "lioness_output", start=1, end = 4, subset_numbers='0,1,2', subset_names = '',save_single=True)
```
Comment (marouenbg) [inline tests/test_lioness.py:80]: why is subst_numbers a string rather than a numpy vector?

Comment (violafanfani) [reply tests/test_lioness.py:80]: good point, old habits

Comment (marouenbg) [reply tests/test_lioness.py:80]: Cool. Which one takes precedence when both START/END and subset_numbers are given?

Comment (violafanfani) [reply tests/test_lioness.py:80]: Ok, now it takes a list of integers of sample names as input. The precedence is given to the subsets (I have updated the documentation). If no subset is specified, then start and end are used (by default we always have start=1 and end=None).

**Pull request merged**